packages/rpm/hazelcast.spec

```diff
@@ -0,0 +1,51 @@
+%define hzversion 4.1-BETA-1
```

Comment:
Shouldn't we be including the last GA version instead?

Reply:
We will update it when 4.1 GA is released.